README.rst

```diff
@@ -893,6 +898,9 @@ There are lots of other projects, which allow you to run your own
 PyPI server. If ``pypiserver`` doesn't work for you, the following are
 among the most popular alternatives:
 
+- `warehouse`_:
+  the software that powers PyPI_ itself.
```

Comment:
warehouse is really not meant to be used for anything other than pypi.org. It has a lot of things that are very specific toward being pypi.org; and nothing else. I don't think warehouse should be suggested as an alternative. :)

Reply:
Yeah, good, glad you mentioned that. I wanted to explicitly say it here, but I notice this section being used for things even beyond alternatives (e.g. it has twine in it which has nothing to do with being an index server at all) -- @mplanchard do you think it's worth splitting the section entirely into "alternatives" and "related software" (and then warehouse goes in the second, yeah @pradyunsg?)?

Reply:
Yup!